### PR TITLE
feat: add new features in shortcut

### PR DESCRIPTION
**Pull request by vromeiro**
## Description

Add new features in shortcut

Fixes # (issue)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

## Screenshots (if appropriate):

Simulator Screen Shot - iPhone 14 Pro Max - 2023-01-25 at 15 38 29

## Checklist:

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) document
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have made corresponding changes to the documentation (if appropriate)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] All new and existing tests passed

### CODE REVIEW

File: OceanComponents/Classes/Components/Shortcut/Ocean+Shortcut.swift [hunk 1]
```diff
@@ -80,9 +101,9 @@ extension Ocean {
 
             NSLayoutConstraint.activate([
                 carouselCollectionView.topAnchor.constraint(equalTo: topAnchor),
-                carouselCollectionView.leftAnchor.constraint(equalTo: leftAnchor),
-                carouselCollectionView.rightAnchor.constraint(equalTo: rightAnchor),
-                carouselCollectionView.heightAnchor.constraint(equalToConstant: Constants.height)
+                carouselCollectionView.leadingAnchor.constraint(equalTo: leadingAnchor),
```
Comment (AcassioM) [inline OceanComponents/Classes/Components/Shortcut/Ocean+Shortcut.swift:104]: vc nao quis usar o setContraints?

File: OceanDesignSystem/Controllers/Components/ShortcutViewController.swift [hunk 2]
```diff
@@ -38,20 +56,27 @@ final public class ShortcutViewController : UIViewController {
...
+            Ocean.ShortcutModel(image: Ocean.icon.documentOutline!, badgeNumber: 3, title: "Label", subtitle: "Label subtitle"),
+            Ocean.ShortcutModel(image: Ocean.icon.documentOutline!, badgeNumber: 0, badgeStatus: .neutral, title: "Label", subtitle: "Label subtitle")
+        ], quantityPage: 2)
     }
     
     private func add(view: UIView) {
         self.view.addSubview(view)
         view.translatesAutoresizingMaskIntoConstraints = false
         
         NSLayoutConstraint.activate([
```
Comment (AcassioM) [inline OceanDesignSystem/Controllers/Components/ShortcutViewController.swift:75]: aqui tbm?

**Review by AcassioM — APPROVED** [review]

File: OceanComponents/Classes/Components/Shortcut/Ocean+Shortcut.swift (hunk 1 above)
Comment (vromeiro) [reply OceanComponents/Classes/Components/Shortcut/Ocean+Shortcut.swift]: opa, feito

File: OceanDesignSystem/Controllers/Components/ShortcutViewController.swift (hunk 2 above)
Comment (vromeiro) [reply OceanDesignSystem/Controllers/Components/ShortcutViewController.swift:75]: esse é no app de exemplo, depois podemos melhorar e criar um base pra esses exemplos

**Pull request merged**